### PR TITLE
Adding code to recover party_id from party service by email address

### PR DESCRIPTION
**Pull request by oddjobz**
Adds a simple lookup to recover a party_id from the party service based on the requested login id (email address) and inserts that party_id into the JWT in place of the previously hard-coded id.

### CODE REVIEW

**Comment by oddjobz** [comment]
Looking at at failing test - needs a new mocked endpoint

File: app/views/sign_in.py [hunk 1]
```diff
@@ -100,7 +116,7 @@ def login():
             "username": username,
             "user_uuid": "ce12b958-2a5f-44f4-a6da-861e59070a32",
```
Comment (rjweeks) [inline app/views/sign_in.py]: So we can delete the user_uuid now?

File: app/views/sign_in.py [hunk 2]
```diff
@@ -100,7 +116,7 @@ def login():
             "username": username,
```
Comment (rjweeks) [inline app/views/sign_in.py]: Do we need username?

File: app/views/sign_in.py [hunk 3]
```diff
@@ -100,7 +116,7 @@ def login():
...
             "user_uuid": "ce12b958-2a5f-44f4-a6da-861e59070a32",
             "role": "respondent",
-            "party_id": "db036fd7-ce17-40c2-a8fc-932e7c228397"
+            "party_id": party_id    #"db036fd7-ce17-40c2-a8fc-932e7c228397"
```
Comment (rjweeks) [inline app/views/sign_in.py]: Delete the hardcoded id in the comment.

**Comment by rjweeks** [comment]
Need to remove user_uuid and username from token later.

File: app/views/sign_in.py (hunk 3 above)
Comment (oddjobz) [inline app/views/sign_in.py]: (deleted)

File: tests/test_surveys.py [hunk 4]
```diff
@@ -109,23 +114,25 @@ def test_access_survey_page(self, mock_object):
...
 
         survey_params = {
             'case_id': case_id,
             'collection_instrument_id': collection_instrument_id
         }
-
         response = self.app.post('/surveys/access_survey', headers=self.headers, data=survey_params)
         self.assertEqual(response.status_code, 200)
 
+        print("Response>", response)
```
Comment (andrewmil) [inline tests/test_surveys.py]: Should this be here?

Comment (oddjobz) [inline tests/test_surveys.py]: Nope. (removed)

**Pull request merged**